griptape/configs/logging/json_formatter.py

```diff
@@ -0,0 +1,19 @@
+import json
+import logging
+from typing import Any
+
+from attrs import define, field
+
+
+@define
+class JsonFormatter(logging.Formatter):
+    indent: int = field(default=2, kw_only=True)
+
+    def __attrs_pre_init__(self) -> None:
+        super().__init__()
```

Comment:
does this need to get set on the filters too?

Reply:
Not sure, I just ran into errors without it.